AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4120,6 +4121,42 @@ function Debug-AzStorageAccountEntraKerbAuth {
                 Write-Error $_
             }
 
+        }
+        #
+        #Check if Fiddler Proxy is cleaned up
+        #
+        if (!$filterIsPresent -or $Filter -match "CheckFiddlerProxy")
+        {   
+           try 
+           {
+                $checksExecuted += 1;
+                
+                $ProxysubFolder = Get-ChildItem -Path Registry::HKLM\SYSTEM\CurrentControlSet\Services\iphlpsvc\Parameters\ProxyMgr
+                foreach($folder in $ProxysubFolder)
+                {
+                    $properties = $folder | Get-ItemProperty
+                    if(($null -ne $properties.StaticProxy) -or ($properties.StaticProxy.Contains("https=127.0.0.1:")))
+                    {
+                        $checks["CheckFiddlerProxy"].Result = "Failed"
+                        Write-Error "CheckFiddlerProxy - FAILED"
+                        Write-Error "Fiddler Proxy is set, you need to delete any registry nodes under ${$properties.PSParentPath} "

```

Comment:
```suggestion
                        Write-Error "Fiddler Proxy is set, you need to delete any registry nodes under ${$properties.PSParentPath}. To prevent this issue from re-appearing in the future, you can also uninstall Fiddler."
```